hztb-servicemanager/src/main/java/com/svs/hztb/sm/common/util/JsonUtil.java

```diff
@@ -5,19 +5,22 @@
 import java.io.Writer;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.svs.hztb.common.logging.Logger;
+import com.svs.hztb.common.logging.LoggerFactory;
 
 public class JsonUtil {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-52-23-152-124.compute-1.amazonaws.com:9000/coding_rules#rule_key=squid%3AS1118)